index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porfolio project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <header class="header-section">
+    <a class="logo">My Logo</a>
+    <nav class="navigation">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+    </nav>
+    </header>
+    <section class="main-section">
+        <h1>I'm Therese TUYISABE<br> Glad to see you</h1>
+        <div class="primary text">
+            <p class="text1">
+            I am software developer!I can help you build a product feature or 
+            website lookthrough sameof my work and experience!
+            if you like what you see and have a project you need coded don't hestiate to contact me.
+            </p>
+            <a class="text2">LET'S CONNECT</a>
+        </div>
+        <ul class="social-icon">
+            <li><a href="index.html"><img src="images/1.png" alt="Twitter"/></a></li>
+            <li><a href="index.html"><img src="images/2.png" alt="Linkedin"/></a></li>
+            <li><a href="index.html"><img src="images/3.png" alt="Medium"/></a></li>
+            <li><a href="index.html"><img src="images/4.png" alt="Github"/></a></li>
+            <li><a href="index.html"><img src="images/5.png" alt="Angellist"/></a></li>
+        </ul>
+        </section>
+<div class="grid-container">
```

Comment:
- [ ] KIndly ensure you align the content of the section of your work in a `<section>` tag because the works section is a standalone content which is the essence of using a semantic `<section>` tag as it is used to group contents that are standalone 👍🏽